lib/networkInjection.js

```diff
@@ -0,0 +1,137 @@
+// This function will be injected into the target process.
+module.exports = function(require, debug, options) {
+  var url = require('url');
+  var STATUS_CODES = require('http').STATUS_CODES || {};
+
+  var lastRequestId = 0;
+
+  function _makeStackTrace(fn) {
```

Comment:
`ConsoleAgent` uses a magical offset, might be worth to use the `fn` argument there as well instead.